Lib/defcon/objects/layer.py

```diff
@@ -223,14 +224,13 @@ def insertGlyph(self, glyph, name=None):
         # it is crucially important that the data be reconstructed
         # in its entirety so that the parent data is properly set
         # in all of the various objects.
-        from copy import deepcopy
+        # FIXME: Please Explain!!
```

Comment:
The layer to layer example I used what just a quick thought. A much more common use case is moving glyphs from font to font. The parenting system would go haywire if a glyph could have > 1 parent. Also, I often send a glyph from one environment into another, for example RoboFab -> defcon. That's trippy, but useful.

Your deserialization case is a good example that I never thought of. Copying is unnecessarily expensive in that case. It could work without copying if certain conditions occur:

- there is no parent object.
- the object is an instance of the same class that the layer would use for a new glyph.
- the object does not have a notification dispatcher (and therefore nothing is observing it).

I've opened an issue for this, #42, and we can address it after the merge.